e2e/cypress/integration/ui/settings/04_project_mgmt.spec.ts

```diff
@@ -0,0 +1,206 @@
+    cy.get('[data-cy=edit-id]').click();
+
+    cy.get('#create-id input').should('have.value', ruleID);
+
+    // Verify correct attributes/operators are selectable for each resource
+    // Event
+    cy.get('#create-type-dropdown').select('Event');
+    cy.get('[data-cy=attribute-dropdown]').select('Chef Organization');
+    cy.get('[data-cy=attribute-dropdown]').select('Chef Server');
+    cy.get('[data-cy=operator-dropdown]').select('equals');
+    cy.get('[data-cy=operator-dropdown]').select('member of');
+
+    // Node
+    cy.get('#create-type-dropdown').select('Node');
+    const nodeAttributes = ['Chef Organization', 'Chef Server', 'Environment',
+        'Chef Role', 'Chef Tag', 'Chef Policy Name', 'Chef Policy Group'];
+
+    nodeAttributes.forEach((att: string) => {
+        cy.get('[data-cy=attribute-dropdown]').select(att);
+    });
+
+    cy.get('[data-cy=operator-dropdown]').select('member of');
+    cy.get('[data-cy=operator-dropdown]').select('equals');
+    cy.get('[data-cy=rule-value] input').type('Chef Policy Group Foo');
+
+    cy.get('#right-buttons button').contains('Save Rule').click();
+    cy.get('chef-page').should('not.be.visible');
+
+    cy.url().should('include', `/settings/projects/${projectID}`);
+    cy.get('chef-td').contains(ruleID);
+  });
+
+  it('can see a list of rules for a project', () => {
+    ['Name', 'ID', 'Resource Type', 'Conditions', 'Edits'].forEach((header) => {
+      cy.get('chef-th').contains(header);
+    });
+
+    cy.get('chef-td').contains(ruleID);
+    cy.get('chef-td').contains(ruleName);
+    cy.get('chef-td').contains('Node');
+    cy.get('chef-td').contains('1 condition');
+    cy.get('chef-td').contains('Edits pending');
+  });
+
+  it('can update a project rule', () => {
+    const updatedRuleName = `updated ${ruleName}`;
+    cy.get('a').contains(ruleName).click();
+
+    cy.url().should('include', `/settings/projects/${projectID}/rules/${ruleID}`);
+    cy.get('chef-page').should('be.visible');
+
+    cy.get('#create-name input').clear().type(updatedRuleName, { delay: 50 })
+      .should('have.value', updatedRuleName);
+
+    cy.get('#create-id input').should('have.value', ruleID);
+    cy.get('#create-id input').should('be.disabled');
+
+    cy.get('#create-type-dropdown option:first').should('have.value', 'NODE');
+    cy.get('#create-type-dropdown option:first').should('be.selected');
+    cy.get('#create-type-dropdown').should('be.disabled');
+
+    cy.get('chef-button').contains('Add Condition').click();
+    cy.get('.end-row-items span').contains('AND').should('be.visible');
+
+    cy.get('[data-cy=attribute-dropdown]:last')
+      .select('Environment');
+
+    cy.get('[data-cy=operator-dropdown]:last')
+      .select('member of');
+
+    cy.get('[data-cy=rule-value]:last input')
+      .type('Dev, Prod, Test');
+
+    cy.get('#right-buttons button').contains('Save Rule').click();
```

Comment:
Consider testing that the Save Rule button is, in fact, enabled.

Reply:
Cypress takes care of that with [actionability checks](https://docs.cypress.io/guides/core-concepts/interacting-with-elements.html#Actionability) on every `click`

Reply:
Nice. 👍